nextflow_schema.json

```
@@ -68,6 +68,119 @@
+            "default": "",
+            "properties": {
+                "run_taxa_classification": {
+                    "type": "boolean",
+                    "description": "Activates the taxonomic classification of input mucleotide sequences.",
+                    "help_text": "This flag turns on the taxonomic classification of input nucleotide sequences. The taxonomic annotations should be turned on if the input metagenomes' bacterial sources are unknown, which can help identify the source of the AMP, BGC or ARG hit obtained for laboratory experiments. This flag should be turned off (which is by deafult) if the input nucleotide sequences represnet a single known genome or *nf-core/mag* was run beforhand. Turning on this flag relatively decreases the pipleine speed and requires >8GB RAM. Due to the size of the resulting table, the final complete summary is in a zipped format.",
+                    "fa_icon": "fas fa-ad"
+                },
+                "taxa_classification_tool": {
+                    "type": "string",
+                    "default": "mmseqs2",
+                    "help_text": "This flag specifies which tool for taxonomic classification should be activated. At the moment only 'MMseqs2' is incorporated in the pipeline.",
+                    "description": "Specifies the tool used for taxonomic classification.",
+                    "fa_icon": "fas fa-check-circle"
+                }
+            },
+            "fa_icon": "fas fa-address-book"
+        },
+        "taxonomic_classification_mmseqs2_databases": {
+            "title": "Taxonomic classification: MMseqs2 databases",
+            "type": "object",
+            "description": "These parameters influence the database to be used in classifying the taxonomy.",
+            "default": "",
+            "properties": {
+                "taxa_classification_mmseqs_databases_localpath": {
+                    "description": "Specify a path to MMseqs2-formatted database.",
+                    "help_text": "Specify a path to a database that is prepared in MMseqs2 format as detailed in the [documentation](https://mmseqs.com/latest/userguide.pdf).",
+                    "fa_icon": "fab fa-stackpath"
+                },
+                "taxa_classification_mmseqs_databases_id": {
+                    "type": "string",
+                    "default": "Kalamari",
+                    "help_text": "Specify which MMseqs2-formatted database to use to classify the input contigs. This can be a nucleotide or amino acid database that includes taxonomic classifications. For example, both GTDB (an amico acid database) and SILVA (a nucleotide database) are supported by MMseqs2. More details can be found in the [documentation](https://mmseqs.com/latest/userguide.pdf).\\n\\n>  Modifies tool parameter(s):\\n> - mmseqs databases <third positional argument>\".",
```

```suggestion
                    "help_text": "Specify which MMseqs2-formatted database to use to classify the input contigs. This can be a nucleotide or amino acid database that includes taxonomic classifications. For example, both GTDB (an amico acid database) and SILVA (a nucleotide database) are supported by MMseqs2. More details can be found in the [documentation](https://mmseqs.com/latest/userguide.pdf).\\n\\n>  Modifies tool parameter(s):\\n> - mmseqs databases <name>\".",
```

Is maybe more clear.